features/authentication/contracts/simple_sign_up.md

```diff
@@ -0,0 +1,44 @@
+# Simple sign up
+
+## Sign up [/auth/sign-up]
+
+This resource represents a basic sign up process.
+
+### Sign up [] [POST] 🔓
+
+The password should follow the following regex:
+```regexp
+/^(?=.{8,}$)(?=.*[A-Z])(?=.*[a-z])(?=.*[0-9]).*/
+```
+
+* **Body**
+  ```json
+  {
+    "name": "Foo",
+    "surname": "Bar",
+    "email": "foo@mrmilu.com",
+    "password": "Secret123&"
+  }
+  ```
+
++ **Response 200** (application/json)
+  Back-end sends validation email to user and response to client.
+  ```json
+  {
+    "token": "foobartoken"
+  }
+  ```
+
+### Sign up validation [/validate] [POST] 🔒
+
+User opens link from email with token. Front-end opens link and retrieves
+token from it for sign up validation.
+
+* **Body**
+  ```json
+  {
+    "token": "validationToken"
+  }
+  ```
+
++ **Response 204**
```

Comment:
I'd like to define the status codes and error codes. I assume the validation token has expiration, which probably should be informed / exposed (jwt?).

Reply:
Yes, we have to discuss the periodicity if the token and how to handle revalidation if we are going to do that. And +1 on showing error responses for those we intend to be handled.